packages/react-components/src/components/gif.tsx

```diff
@@ -235,9 +239,8 @@ const Gif = ({
             onMouseLeave={onMouseLeave}
             onClick={onClick}
             onContextMenu={(e: SyntheticEvent<HTMLElement, Event>) => onGifRightClick(gif, e)}
-            ref={container}
         >
-            <div style={{ width, height, position: 'relative' }}>
+            <div style={{ width, height, position: 'relative' }} ref={container}>
```

Comment:
@thighhighcrocs separate from this PR, but why do I have this container? Is the width and height redundant?

Reply:
A bit, but now that the `ref` moves onto this object I think it's necessary. I did this because the changing of the wrapper component was breaking the `ref` and causing the intersection observer to stop working. Moving it to this element, since it stays, fixed the issue.

Reply:
@thighhighcrocs oh man, right, we'd need to forward the ref